src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -153,6 +153,23 @@ private void recordNewSentEmail(String subject,
 
         moveJobToGroupModeTaskQueue();
         moveJobToAddressModeTaskQueue();
+
+        if (addressModeOn && groupModeOn) {
+            statusToAdmin = "Email will be sent within an hour to "
```

Comment:
`statusToAdmin` will be the log message. Log message is for us to debug when something bad happens. Thus, we need to put something useful like `Email Queued: ...` (So we could know the email is queued.)